server/src/main/java/io/druid/server/coordinator/rules/LoadRule.java

```diff
@@ -57,12 +59,58 @@ public CoordinatorStats run(DruidCoordinator coordinator, DruidCoordinatorRuntim
 
     final Map<String, Integer> loadStatus = Maps.newHashMap();
```

Comment:
@xanec could you please extract some functionality into dedicated methods from this `run()` method. It's too long.

Reply:
Sure, do you mind if I do some significant significant refactoring?

Reply:
I don't mind